src/applications/vaos/appointment-list/components/AppointmentsPage/AppointmentCard.jsx

```diff
@@ -68,7 +69,7 @@ export default function AppointmentCard({
     }appointment on ${appointmentDate.format('dddd, MMMM D h:mm a')}`;
   };
   const VAFacilityName = () => {
-    if (facility) {
+    if (facility && !isEmpty(facility)) {
```

Comment:
It looks like facility is defaulted to an empty object so this resolves to an empty `<></>` in those cases. Looking at the logic I think the intention is to fallback to `VA appointment` in that case so I added this check. Let me know if this is dangerous since it's a functional behavior change.